print-bgp.c

```diff
@@ -2766,6 +2795,15 @@ bgp_capabilities_print(netdissect_options *ndo,
                 cap_offset += 4;
             }
             break;
+        case BGP_CAPCODE_ROLE:
+            if (cap_len < 1) {
+                ND_PRINT(" (too short, < 1)");
+                return;
+            }
+            ND_PRINT("\n\t\t Role name %s",
```

Comment:
Comparing to other capabilities, there's no space between the `\n\t\t` and the description.  Note the additional indentation of the "Role name" vs. "AFI" in the test output:
```
	    Option Capabilities Advertisement (2), length: 3
	      Role Capability (9), length: 1
		 Role name Customer
	    Option Capabilities Advertisement (2), length: 6
	      Multiple Paths (69), length: 4
		AFI IPv4 (1), SAFI Unicast (1), Send/Receive: Receive
```

Reply:
Thanks, fixed.